src/lib.rs

```diff
@@ -5,59 +5,76 @@
 #[macro_use]
 extern crate alloc;
 
-extern crate parity_wasm;
 extern crate byteorder;
-#[macro_use] extern crate log;
+extern crate parity_wasm;
+#[macro_use]
+extern crate log;
 
 pub mod rules;
 
 mod build;
-mod optimizer;
-mod gas;
-mod symbols;
 mod ext;
+mod gas;
+mod optimizer;
 mod pack;
 mod runtime_type;
+mod symbols;
 
 pub mod stack_height;
 
-pub use build::{build, SourceTarget, Error as BuildError};
-pub use optimizer::{optimize, Error as OptimizerError};
+pub use build::{build, Error as BuildError, SourceTarget};
+pub use ext::{
+    externalize, externalize_mem, shrink_unknown_stack, underscore_funcs, ununderscore_funcs,
+};
 pub use gas::inject_gas_counter;
-pub use ext::{externalize, externalize_mem, underscore_funcs, ununderscore_funcs, shrink_unknown_stack};
+pub use optimizer::{optimize, Error as OptimizerError};
 pub use pack::{pack_instance, Error as PackingError};
 pub use runtime_type::inject_runtime_type;
 
-pub struct TargetRuntime {
-	pub create_symbol: &'static str,
-	pub call_symbol: &'static str,
-	pub return_symbol: &'static str,
+pub struct TargetSymbols {
+    pub create: &'static str,
+    pub call: &'static str,
+    pub return_: &'static str,
```

Comment:
btw don't like this `_` at the end of the identifier, especially in the public api `ret` would be

Reply:
#114